Part2

```diff
@@ -0,0 +1,34 @@
+Matthew Danielson
+7/7/16
+Part 2 of Assignment7
+
+1) ++*p increments the value of a pointer, and *p++ increments the address of a pointer. *++p is not valid, as it is incrementing invalidly.
+
+2) Left to right order is guaranteed for operator precedence; without this, many operators become essentially useless, because the logic of the code would not be reliable.
+
+3) Pointers are passed by reference allowing for multiple return types. Additionally, constant pointers can be made; additionally, arrays of pointers allow for non-standard char arrays that are of varying size. They can also be used to return an array of values.
+
+4) a[0] = 11
+a[1] = 11
+a[2] = 12
+a[3] = 13
+a[4] = 14
+a[5] = 15
+a[6] = 16
+a[7] = 17
+a[8] = 18
+a[9] = 19
+p = 12
+char *varname[]
+4.1 char[]
+4.2 would subtract ascii values, resulting in value 1, or SOH
```

Comment:
Close!  "xyz" is just an array of characters, so the "xyz"[1] is just accessing "y". Then you just subtract 'y' from it to get 0. This is because, if you can recall, a char is just a value mapped to a character!